tests/TestCase/Controller/Component/CrudComponentTest.php

```diff
@@ -120,6 +121,16 @@ public function index()
     {
         return $this->Crud->execute('index');
     }
+
+    /**
+     * Test it should render 'list'
+     *
+     * @return void
+     */
+    public function listing()
```

Comment:
Since `list` is a reserved word I had to rename this method. Makes me wonder if `ListAction` should be renamed completely.